### ISSUE TITLE
[Bug] Regression from 1.1.1223

### ISSUE THREAD

**Issue by Littl3Guy** [Ryujinx/Ryujinx#6498]
### Description of the issue

Various objects flickering in The Legend of Zelda: Tears of the Kingdom.

### Reproduction steps

Panning the camera around while looking at certain objects.

### Log file

[Ryujinx_1.1.1223_2024-03-14_22-39-45.log](https://github.com/Ryujinx/Ryujinx/files/14610237/Ryujinx_1.1.1223_2024-03-14_22-39-45.log)

### OS

Windows 11

### Ryujinx version

1.1.1223

### Game version

1.2.1

### CPU

R7 3800X

### GPU

RX 6700XT

### RAM

16GB

### List of applied mods

None.

### Additional context?

Here's a couple examples:

https://github.com/Ryujinx/Ryujinx/assets/115218507/e9671860-33ff-412f-8be7-b6eefa7807d3

https://github.com/Ryujinx/Ryujinx/assets/115218507/ac8c0f70-58dc-4f22-8ce1-8ace393a8c37

**Comment by Anthonyc833** [comment]
does this happen when youre close up and on the ground aswell?

**Comment by Littl3Guy** [comment]
> does this happen when youre close up and on the ground aswell?

Yeah it happens pretty much everywhere, these were just easier for me to capture.

**Comment by Anthonyc833** [comment]
so its 1005 a rendering issue but im not sure about the exact script that would be responsible for it because the log doesnt directly mention if it errors

**Comment by NikkiTanaka** [comment]
Can be partially fixed on any later version of Ryujinx by downgrading the game to 1.2.0 and applying Disable Quality Reduction and LOD Improvement mods (or maybe either of them).

The thing is, it doesn't fix flickering NPC models, which happens from time to time (carriages, for example, even cover slightly more distance the moment they disappear and reappear).

**Comment by BooBerry** [comment]
This issue still exists with the latest build of Ryujinx. With the recent fix to RDNA3 GPUs with Vulkan, this is (as far as I know) the only remaining (major?) TOTK issue when using AMD GPUs. Using Windows 11 and Ubuntu 24.04.1 LTS with a AMD 7900X CPU and 7800 XT GPU.

Any chance this could be looked into? I would assume this exists with platforms like the Steam Deck and ROG Ally too.

**Comment by riperiperi** [comment]
It doesn’t appear to happen on the steam deck, or it would be fix. I can only reliably fix the issue if I can reproduce it.

**Comment by jucasleiva** [comment]
This issue is too anooying when you move your camera or move away get that flickering with for exemple that dragon in begin of the game and the robots. My config:

CPU
Ryzen 5 5600G

GPU
RX 6600XT

RAM
32GB

Ryujinx version
1.1.1385

**Comment by shindio** [comment]
1.0.0 is normal

CPU
10700k

GPU
RX 6600XT     24.7.1

RAM
32GB

Ryujinx version
1.1.1385

**Comment by antoniodesousa** [comment]
I also have the flickering issue pretty much everywhere.

https://github.com/user-attachments/assets/83774532-110e-4d79-9345-e6f87274ee3b

Ryujinx version 1.1.1385
[Ryujinx_1.1.1385_2024-09-03_11-57-27.log](https://github.com/user-attachments/files/16847138/Ryujinx_1.1.1385_2024-09-03_11-57-27.log)

CPU Ryzen 9 6900HK
GPU Radeon RX 6600M 24.8.1
RAM 64GB
OS Windows 11 Pro 23H2

**Comment by brolaru** [comment]
Currently having this same problem (LoD and NPC flickering) on 1.2.1 but had it since the game first released

OS 
Microsoft Windows 10 IoT Enterprise LTSC

CPU
Ryzen 5 5600

GPU
RX 6650XT

RAM
16GB

Ryujinx version
1.1.1385

**Comment by jucasleiva** [comment]
It's as if depending on the distance it causes this flickering

https://github.com/user-attachments/assets/b51c1dc8-4016-49c2-90cc-e580aa83b626

https://github.com/user-attachments/assets/014c7436-cade-4cc1-899f-e68b741aee6e
 
CPU: Ryzen 5600g
GPU: Rx 6600xt
Windows 11 23H2
